website/docs/state/workspaces.html.md

```diff
@@ -39,6 +39,13 @@ It was renamed in 0.10 based on feedback about confusion caused by the
 overloading of the word "environment" both within Terraform itself and within
 organizations that use Terraform.
 
+-> **Note**: The Terraform CLI workspace concept described in this document is
+different from but related to the Terraform Cloud
+[workspace](https://www.terraform.io/docs/cloud/workspaces/index.html) concept.
```

Comment:
@nfagerlund :  Would your proposed change really work?  This PR is for hashicorp/terraform while the Terraform Cloud Workspace documentation is in hashicorp/terraform-website which pulls in this content as a git module.  If you're sure it will work, I trust you and will accept it.

Reply:
Yup: it all gets built into the same website on the same domain, and the path resolution is handled by the web browser, so it works as expected. Using plain paths also makes it so links work correctly when you're viewing a local preview. 

I realize it seems a bit squirrelly with stuff in separate repos; sorry. IMO, it's easiest to reason about if you think of the content split as being solely for the convenience of the devs, rather than as being semantic.